packages/opds-web-client/src/app.tsx

```diff
@@ -23,12 +25,12 @@ class OPDSWebClient {
   }, elementId: string) {
     this.elementId = elementId;
     this.pathPattern = config.pathPattern || "/(collection/:collectionUrl/)(book/:bookUrl/)";
-    this.RouteHandler = React.createClass({
+    this.RouteHandler = createReactClass({
```

Comment:
Any plans on moving this soon to the `React.component` pattern?

Reply:
We can if you want :)

I think this is the only place it's used.

Reply:
Might be good to also remove the `react-create-class` dependency.

Reply:
Agreed.